server/app-engine/gtfsrt.py

```diff
@@ -418,6 +439,46 @@ def add_alert(datastore_client, alert):
     datastore_client.put(entity)
     print('+ wrote alert')
 
+def delete_alert(datastore_client, alert):
+    print('delete_alert()')
+    print('- alert: ' + str(alert))
+
+    query = datastore_client.query(kind='alert')
+    # We need to filter for all fields, since a unique ID isn't passed into the protobuf
+    query.add_filter('agency_key', '=', alert["agency_key"])
+    query.add_filter('cause', '=', alert["cause"])
+    query.add_filter('effect', '=', alert["effect"])
+    query.add_filter('description', '=', alert["description"])
+    query.add_filter('time_start', '=', alert["time_start"])
+    query.add_filter('time_stop', '=', alert["time_stop"])
+    if(alert["stop_id"] != ""):
```

Comment:
Without having seen all of your changes, I will assume that delete_alerts is a public interface. How do you know that clients will set unused fields to the empty string, vs. `None` or something else?